### PR TITLE
added isCutting prop to Heading

### PR DESCRIPTION
**Pull request by aminbksi**
https://momentum.nifty.pm/l/u9CurUSFiv

subtask: IsCutting prop of the Heading component

### CODE REVIEW

File: src/scenes/collaboration/pages/DashboardPage/components/templates/Dashboard/components/TileItem/components/TileDetail.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ const TileDetail: FC<PropsInterface> = ({
       title={tile.content?.title ?? ''}
       noPadding={!tile.content?.text}
       headerStyle="uppercase"
-      headerClassName="header-eclipse"
+      isCuttingHeader={true}
```
Comment (dmitry-yudakov) [inline src/scenes/collaboration/pages/DashboardPage/components/templates/Dashboard/components/TileItem/components/TileDetail.tsx:48]: `={true}` part is not needed here

File: src/scenes/widgets/pages/MusicPlayerWidget/MusicPlayerWidget.tsx [hunk 2]
```diff
@@ -35,11 +35,11 @@ const MusicPlayerWidget: FC = () => {
       titleWidth="145px"
       headerItem="center"
       headerType="h4"
+      isCuttingHeader={true}
```
Comment (dmitry-yudakov) [inline src/scenes/widgets/pages/MusicPlayerWidget/MusicPlayerWidget.tsx:38]: and here

File: src/ui-kit/atoms/Heading/Heading.styled.ts [hunk 3]
```diff
@@ -56,21 +56,51 @@ export const Heading = styled.div`
...
+    width: 100%;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
   font-size: var(--font-size-s);
 `;
 
 export const H4 = styled.h4`
+  &.header-eclipse {
```
Comment (PAKovalev) [inline src/ui-kit/atoms/Heading/Heading.styled.ts:89]: You can define this class under `main.scss` btw.

**Review by PAKovalev — APPROVED** [review]

File: src/ui-kit/molecules/ToastContent/ToastContent.tsx [hunk 4]
```diff
@@ -34,7 +34,7 @@ const ToastContent: FC<PropsInterface> = (props) => {
         componentSize={{width: '100%'}}
         titleWidth="370px"
         headerType="h4"
-        headerClassName="header-eclipse"
+        isCuttingHeader={true}
```
Comment (dmitry-yudakov) [inline src/ui-kit/molecules/ToastContent/ToastContent.tsx:37]: and here

**Review by dmitry-yudakov — APPROVED** [review]

**Pull request merged**